### DISCUSSION TITLE
Different jira server for API and for UI

### DISCUSSION THREAD

**Discussion by krzyk** [ankitpokhrel/jira-cli#604] [Ideas]
Our corporate jira uses different jira server for API access and diffrent for browser access.
So using enter on given issue opens up browser with http error, because we can't the API server in browser.

Example.
API server: jira-api.example.com
UI server: jira.example.com

**Comment by markhatch** [comment]
We have the same thing with our on-prem install.
I'm playing around with adding an optional config option called "view_server" and adding a helper function "GenerateServerURL". The helper function just replaces the `server` when writing any of the `\browse\` URLs generated. Appears to work in my local testing. Let me know if I should try opening a PR.

**Comment by ankitpokhrel** [comment]
Will look into this, PRs are welcome in the meantime :)

**Reply by markhatch** [reply]
Generated a PR here https://github.com/ankitpokhrel/jira-cli/pull/610

**Comment by ankitpokhrel** [comment]
This is done with #613 (thanks @markhatch) and is available in the new release [v1.4.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.4.0)
